devcontainer/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ros:humble-ros-base-jammy
```

Comment:
Looks good but please rename the directory from `devcontainer` to `.devcontainer`. It probably doesn't matter (I've seen this named `docker` in other repos) but GitHub says [here](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/introduction-to-dev-containers):

> The configuration files for a dev container are contained in a .devcontainer directory in your repository. 

We may eventually need to enable GitHub codespaces for this repo too, so it's good to address this early.

Reply:
Also, after the rename, you might need to change the reference in `build_docker.yml`

Reply:
Done.